src/utils/index.ts

```diff
@@ -32,7 +32,17 @@ async function manageScreenShots(taskQ: TaskQueuePC, fileNameOrganiser: Set<stri
 	taskQ.runTask(() => new Promise(async (res, rej) => {
 		try {
 			const dialogWindow = WebviewWindow.getByLabel('dialog') as WebviewWindow;
-			await dialogWindow.show();
+
+				// Changing the width and height of the dialog window using setSize.
+				// Note: It will not create a scrollbar.
+
+				// After setting the size, a callback function is call. 
+				// Inside the callback, the decoration is set to true in runtime.
+			  	 dialogWindow.setSize(new LogicalSize(400,200)).then(()=>{
```

Comment:
i dont think it is neceasry to resize window, can u plz remove the resising part
and modify the decorations here